tectonic/synctex.c

```diff
@@ -18,13 +18,13 @@
  *
  * Note: these settings used to depend on the no_pdf_output variable. Tectonic
  * fixes this to 1, but we do then generally run xdvipdfmx to produce a PDF.
- * Need to investigate how to deal with this.
+ * Therefore we fix this for SyncTeX to produce pdf coordinates.
```

Comment:
I think this whole comment paragraph can just be removed now. It was basically a note-to-self that the settings might need to be twiddled to get SyncTeX working, and you figured out the right twiddling!